README.md

```diff
@@ -93,6 +93,38 @@ This is yaml file for configuring gazebo setting.
       imu_sensors_config:
         imu_sensor0: {ros_name: 'ros_imu_sensor', link_name: 'LINK_NAME0', frame_id: 'LINK_NAME0'}
 
+### Custom Plugins
+
+### CranePlugin
+
+This plugin provides fake "Crane" in gazebo environment.
```

Comment:
Please add `To use this plugin, add the following lines to your URDF.` here.

Reply:
I fixed.
Thank you!